@here/harp-mapview/lib/MapView.ts

```diff
@@ -492,7 +492,7 @@ export class MapView extends THREE.EventDispatcher {
      */
     readonly mapRenderingManager: IMapRenderingManager;
 
-    private m_zoomLevelBias: number = 1;
+    private m_zoomLevelBias: number = -1;
```

Comment:
Previously `zoomLevelBias` was multiplier, now you changed it to offset.

I don't have opinion which is good or bad, but it's a change.

Document at least in `get/set` zoomLevelBias that it's offset (or multiplier if you revert logic).

After thining, I think that offset is better than multiplier is hard to control. 
It's just much more easier to reason what is impact of -1, +1 than 0.9, 1.1